index.html

```diff
@@ -5,41 +5,242 @@
+                        <span class="img"> <img src="./images/Counter.jpg" alt="seperation"></span>
+                        <span class="num51">2015</span>
+                    </div>
+                    <p class="to-do-para51">A daily selection of privately personalised reads: no accounts or sign-ups
+                        required.</p>
+                    <div class="works51">
+                        <div class="html51"> <a href="#">
+                                <p class="word3151">html</p>
+                            </a></div>
+                        <div class="css51"><a href="#">
+                                <p class="word2151">css</p>
+                            </a></div>
+                        <div class="jv51"> <a href="#">
+                                <p class="word051">javascript</p>
+                            </a></div>
+                    </div>
+                    <div class="click51">
+                        <button>See Projects</button>
+                    </div>
+                </section>
+                
+                <!-- About myself -->
+                <section class="last">
+                    <h1>About <br> Myself</h1>
+                    <p>Hello, I'm a software developer! I can help you build a product, feature or
+                        website. Look through some of my work and experience! if you like what you see
+                        and have a project you need coded, dont hesitate to contact me.
+                    </p>
+                    <h3>LETS CONNECT</h3>
+                    <ul class="social-media48">
+                        <li> <a href="#"><img src="./images/github.jpg" alt="github"></a> </li>
+                        <li> <a href="#"><img src="./images/medium.jpg" alt="Linkedin"></a> </li>
+                        <li> <a href="#"><img src="./images/twitter.jpg" alt="medium"></a> </li>
+                        <li> <a href="#"><img src="./images/Vector.jpg" alt="twitter"></a> </li>
+                    </ul>
+
+                    <div class="click52">
+                        <button>Get my resume</button>
+                    </div>
+
+                    <div class="names22">
+                        <h4>Languages</h4>
+                        <div class="icon-container">
+                            <img src="./images/chevron-down-solid.svg" alt="">
+                        </div>
+                    </div>
+
+                    <div class="prog1">
+                        <img src="./images/Ellipse-1.jpg" alt="work">
+                        <h5>Javascript</h5>
+                    </div>
+
+                    <div class="prog2">
+                        <img src="./images/Ellipse-12.jpg" alt="work">
+                        <h5>HTML</h5>
+                    </div>
+
+                    <div class="prog3">
+                        <img src="./images/Ellipse-9.jpg" alt="work">
+                        <h5>CSS</h5>
+                    </div>
```

Comment:
Kindly wrap this part with the `ul` tag and change the `div` tags to be `li` tags for semantic purposes.